examples/petsc/include/sphereproblemdata.h

```diff
@@ -44,7 +38,7 @@ static BPData bp_options[6] = {
     .num_comp_x = 3,
     .topo_dim = 3,
     .q_data_size = 1,
-    .q_extra = 1,
+    .q_extra = 2,
```

Comment:
ToDo: the BP spec requires 1 here

Reply:
Thanks, I am not sure when this happened.